ipa-core/src/net/test.rs

```diff
@@ -45,6 +46,38 @@ pub const DEFAULT_TEST_PORTS: Ports = Ports {
     shards: [6000, 6001, 6002],
 };
 
+/// A network with two shards per helper.
+pub const TWO_SHARDS: [Ports; 2] = [
```

Comment:
are we using this to spin up an actual server in unit tests? if so, I don't think we should do it - we have to pick ports randomly

Reply:
We have both tests. I can remove the one using this set.

Reply:
I looked at the change and I don't understand whether we ever create a server with hardcoded ports or no. My line of thinking is that we should never need that - hardcoded ports are only good for config validation, everything else should use ports assigned by Kernel

Reply:
Yes, I modified the code so that we only use specified ports for config unit tests. All the HTTP tests now use OS given ports.